src/components/Text.tsx

```diff
@@ -1,10 +1,27 @@
-/* eslint-disable camelcase */
-import React from 'react';
+import React, {FC, ReactNode} from 'react';
 import PropTypes from 'prop-types';
 import chalk from 'chalk';
 
-const Text = ({bold, italic, underline, strikethrough, children, unstable__transformChildren}) => {
-	const transformChildren = children => {
+interface TextProps {
+	readonly bold?: boolean;
+	readonly italic?: boolean;
+	readonly underline?: boolean;
+	readonly strikethrough?: boolean;
+	readonly unstable__transformChildren?: (children: ReactNode) => ReactNode;
+}
+
+/**
+ * This component can change the style of the text, make it bold, underline, italic or strikethrough.
+ */
+export const Text: FC<TextProps & { children: ReactNode }> = ({
```

Comment:
Same as https://github.com/vadimdemedes/ink/pull/264/files#r399674079.

Reply:
FC includes optional children prop. According to propTypes, children is required so children needs to be defined to make it required. I moved chidren prop into TextProps in https://github.com/vadimdemedes/ink/pull/264/commits/0e678de0de66fe570cfb3830baca325af82191d8